.github/workflows/operate-run-tests.yml

```diff
@@ -11,6 +11,7 @@ on:
 jobs:
   run-test:
     runs-on: ubuntu-22.04
+    timeout-minutes: 90
```

Comment:
I see that on the current PR [the execution took only 12 minutes](https://github.com/camunda/camunda/actions/runs/15275218923/job/42959848752?pr=32570) so maybe it is possible to decrease the hard-coded timeout here (thanks for adding one!) to 30min tops?